api/can_api_definition.py

```diff
@@ -149,6 +158,27 @@ class PredictionTimeseriesRowWithHeader(CANPredictionTimeseriesRow):
 class CovidActNowStateTimeseries(CovidActNowStateSummary):
     timeseries: List[CANPredictionTimeseriesRow] = pydantic.Field(...)
 
+    # pylint: disable=no-self-argument
+    @pydantic.validator('timeseries')
+    def check_timeseries_have_cumulative_test_data(cls, rows, values):
```

Comment:
is this a cls method?

Reply:
yeah all of the validators are classmethods